components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1589 @@
+ + \nabla_\perp \cdot \left(  \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right) 
+& 
++ \left( \nabla_\perp \cdot  {\bf u}  \right) {\bf u}
++ \frac{1}{\tilde h}\left(
+ \left[ {\bf u}\right]_{z=z^{\text{top}}}-\overline{ {\bf u} }^{\tilde z}\right) \, \left[\tilde{w}_{tr} \right]_{z=z^{\text{top}}} 
+ - 
+  \frac{1}{\tilde h}\left(
+ \left[ {\bf u}\right]_{z=z^{\text{bot}}}-\overline{ {\bf u} }^{\tilde z}\right) \, \left[\tilde{w}_{tr} \right]_{z=z^{\text{bot}}} 
+\\ &  =
+-  \overline{  \mathbf{f} \times \mathbf{u} }^{\tilde z} 
+-  \overline{  \nabla_\perp \Phi }^{\tilde z} 
+-  \overline{\alpha}^z \overline{\nabla_{\perp} p}^{z}  
++  \overline{\alpha}^z \overline{ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) }^z 
+$$ (momentum-layered-differential-3)
+
+In addition, the horizontal advection may be rewritten as
+
+$$
+\nabla_\perp \cdot \left( {\bf u} \otimes {\bf u}  \right) 
+&= 
+\nabla_\perp \cdot \left( {\bf u} {\bf u}^T  \right) \\
+&= \left( \nabla_\perp \cdot  {\bf u}  \right) {\bf u}
++  {\bf u} \cdot \nabla_\perp {\bf u} 
+$$ (adv2d-prod)
+
+The term ${\bf u} \cdot \nabla_\perp {\bf u}$ may be replaced with the vector identity
+
+$$
+\begin{aligned}
+{\bf u} \cdot \nabla_\perp{\bf u}
+&= (\nabla_\perp\times {\bf u}) \times {\bf u} + \nabla_\perp\frac{|{\bf u}|^2}{2} \\
+&= \left( \boldsymbol{k} \cdot (\nabla_\perp\times {\bf u})\right)
+\left( \boldsymbol{k} \times {\bf u} \right) + \nabla_\perp\frac{|{\bf u}|^2}{2} \\
+&= \zeta {\bf u}^{\perp} + \nabla_\perp K,
+\end{aligned}
+$$ (advection-identity)
+
+where $\zeta$ is relative vorticity and $K$ is kinetic energy.
+This step separates the horizontal advection into non-divergent and non-rotational components, which is useful in the final TRiSK formulation. 
+
+Now [](#momentum-layered-differential-3) becomes
+
+$$
+ \frac{\partial \overline{  {\bf u} }^{\tilde z} }{\partial t} 
+ +
+   \left(  \zeta + f  \right) {\overline{  {\bf u} }^{\tilde z}}^\perp
+& 
++ \left( \nabla_\perp \cdot  {\bf u}  \right) {\bf u}
++ \frac{1}{\tilde h}\left(
+ \left[ {\bf u}\right]_{z=z^{\text{top}}}-\overline{ {\bf u} }^{\tilde z}\right) \, \left[\tilde{w}_{tr} \right]_{z=z^{\text{top}}} 
+ - 
+  \frac{1}{\tilde h}\left(
+ \left[ {\bf u}\right]_{z=z^{\text{bot}}}-\overline{ {\bf u} }^{\tilde z}\right) \, \left[\tilde{w}_{tr} \right]_{z=z^{\text{bot}}} 
+\\ &  =
+- \nabla_\perp K 
+-  \overline{  \nabla_\perp \Phi }^{\tilde z} 
+-  \overline{\alpha}^z \overline{\nabla_{\perp} p}^{z}  
++  \overline{\alpha}^z \overline{ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) }^z 
+$$ (momentum-layered-differential-4)
+
+Notes from Mark:
+
+1. I don't know what to do with the z-average in the advection term of [](#momentum-layered-differential-2) and here. In the mean time, I'll just skip to the layer-averaged terms.
+2. I don't know what to make of the $\left( \nabla_\perp \cdot  {\bf u}  \right) {\bf u}$ term, and whether it cancels with some of those odd vertical transport terms. 
```

Comment:
Don't you get a clean $u \cdot \nabla u$ when you combine your 2 renaming terms?